### DISCUSSION TITLE
Newbie: Getting HomematicIP devices to be discovered

### DISCUSSION THREAD

**Discussion by tonyvfield** [OpenCCU/OpenCCU#2511] [🇺🇸 Q&A] [answered]
I have loaded the add-on, started it running, but then the next step does not work for me since I am missing some info.
Having gone into the Rasberrymatic area I have the same screen as in the documentation.
In the documentation is the following 
"After having performed the above steps you can now add or configure your HomeMatic/homematicIP devices in a similar way like you would do with a real RaspberryMatic system. Thus, you should be able to import and existing configuration, start to teach-in HomeMatic/homematicIP devices (depending on the RF/Wired hardware you are using), etc.2
Since I have not used the rasberrymatic system before and know almost nothing about this area where are the instructions for "a similar way" 
or how do I get the teach in to work? I get the following message on the screen when I try "no homematicIP gateway was found to be useable......

**Comment by jens-maus** [comment, marked as answer]
> I get the following message on the screen when I try "no homematicIP gateway was found to be useable......

Well, have you checked the requirements, actually? See https://github.com/jens-maus/RaspberryMatic/wiki/Einleitung#voraussetzungen

Thus, for being able to communicate to a HomeMatic/homematcIP device you first need a proper "Gateway" hardware. Installing RaspberryMatic (no matter if installed as HA Addon or on RaspberryPi directly) alone will not provide you anything. Thus, you need either a `HmIP-RFUSB` or `RPI-RF-MOD` device to be connected to your HomeAssistant host system or directly connected to the GPIO (in case your HA host is a RaspberryPi) or via appropriate Adapter PCB like `HB-RF-USB-2` or `HB-RF-ETH`. So just check the hardware requirements for being able to communicate to any HomeMatic/homematicIP devices.

**Comment by tonyvfield** [comment]
Hi thanks for the very quick answer.

I believe I do have everything as previously before I had to reset my google HA I used the previous version and that found and communicated with the devices on the network.


What I did find on the raspberrymatic home page was to be able to manually add in a gateway. Once I rebooted the program it has gone from inactive to nothing in status.

I still get the same result of it saying there is not gateway and yet the gateway is there

https://gyazo.com/0571f2fe57fa52157209fb38ad0284fd
